### PR TITLE
implement loading genesis file in x/multisig package

### LINKED ISSUE #205: Register multisig in genesis file
We should be able to register a multisig contract in the genesis file. This multisig contract can then be used as an owner in other configurations from the genesis, such as the owner of an NFT, or the destination for the collected fees.

~In order to make this usable, we should have a tool that can display the addresses that would be generated for the multisig contracts, so we can use those values in the other items of genesis (as a static analysis tool to help build a genesis file). This can be a simple script for this PR and evaluate if and how it is needed to extend later.~

### PR DESCRIPTION
**Pull request by husio**
Allow to load multisig contracts from a genesis file.

New contracts must be defined under `multisig` key. See the init test
file for a complete example.

resolve #205

### CODE REVIEW

File: x/multisig/init_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	var ini Initializer
+	if err := ini.FromGenesis(opts, db); err != nil {
+		t.Fatalf("cannot load genesis: %s", err)
+	}
+
+	bucket := NewContractBucket()
+	obj, err := bucket.Get(db, seq(1))
+	if err != nil {
+		t.Fatalf("cannot fetch contract information: %s", err)
+	} else if obj == nil {
```
Comment (alpe) [inline x/multisig/init_test.go]: pp: no else before the if required. `t.Fatal` ends the execution already

File: x/multisig/init_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		t.Fatal("contract information not found")
+	}
+	if _, ok := obj.Value().(*Contract); !ok {
+		t.Errorf("invalid object stored: %T", obj)
+	}
```
Comment (alpe) [inline x/multisig/init_test.go]: req: please check that the values you have persisted match the source data.

File: x/multisig/model.go [hunk 3]
```diff
@@ -69,3 +69,10 @@ func (b ContractBucket) Save(db weave.KVStore, obj orm.Object) error {
 	}
 	return b.Bucket.Save(db, obj)
 }
+
+// Build assigns an ID to given contract instance and returns it as an orm
+// Object. It does not persist the escrow in the store.
+func (b ContractBucket) Build(db weave.KVStore, c *Contract) orm.Object {
```
Comment (alpe) [inline x/multisig/model.go]: 👍 
q: Was `Build` supposed to save the entity?

File: x/multisig/cmd/lsc/lsc.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
+package main
```
[the remark this answers is not in the document]
Comment (husio) [reply x/multisig/cmd/lsc/lsc.go]: I will move it.

I can add readme, but I thought that adding good `-help` output might be even better. Should I copy `-help` output to README anyway?

File: x/multisig/model.go (hunk 3 above)
Comment (husio) [reply x/multisig/model.go]: Yes, this is almost like `Create` but without saving. It's used in [escrow](https://github.com/iov-one/weave/blob/master/x/escrow/model.go#L163) as well.

**Comment by husio** [comment]
@alpe please check again, I have applied all changes are requested.

File: x/multisig/init_test.go [hunk 5]
```diff
@@ -55,3 +76,12 @@ func seq(val int64) []byte {
 	binary.BigEndian.PutUint64(b, uint64(val))
 	return b
 }
+
+func fromHex(t *testing.T, s string) []byte {
+	t.Helper()
```
Comment (alpe) [inline x/multisig/init_test.go]: 👍

File: x/multisig/init_test.go [hunk 6]
```diff
@@ -41,12 +45,29 @@ func TestGenesisKey(t *testing.T) {
...
 		t.Errorf("invalid object stored: %T", obj)
 	}
+	if want, got := int64(2), c.ActivationThreshold; want != got {
+		t.Errorf("want activation threshold %d, got %d", want, got)
+	}
+	if want, got := int64(2), c.AdminThreshold; want != got {
+		t.Errorf("want admin threshold %d, got %d", want, got)
+	}
+	wantSigs := [][]byte{
+		fromHex(t, "e4c7e4c71a3b301a2521753ddd1d2c26fd6fe1bf"),
```
Comment (alpe) [inline x/multisig/init_test.go]: nice

**Pull request merged**